docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -3505,7 +3505,7 @@ function mass_theme_preprocess_node_news_full(&$variables) {
       if (count($body_array) > 1) {
         $body_array[1] = $location . $body_array[1];
       }
-      $body_string = implode($body_array, '<p>');
+      $body_string = implode('<p>', $body_array);
```

Comment:
@Javi-er imploding an array this way will create many `<p>` open tags. I'm not sure what this code is doing exactly, but the HTML inside `$body_string` will not be correct I think. Would you explain it to me?

Reply:
hi @beto-aveiga This was related to a warning that was showing up, I'm not sure what this code does since it's not written to me, but starting with PHP 7.4 [Passing the separator after the array has been deprecated ](https://www.php.net/manual/en/function.implode.php)and it will not longer be supported starting with PHP 8, the change you see above is just for preventing the warning.